cmd/calico-node/main.go

```diff
@@ -96,7 +97,7 @@ func main() {
 		fmt.Println(startup.VERSION)
 		os.Exit(0)
 	} else if *runFelix {
-		felix.Run("/etc/calico/felix.cfg")
+		felix.Run("/etc/calico/felix.cfg", buildinfo.GitVersion, buildinfo.GitRevision, buildinfo.BuildDate)
```

Comment:
I don't think this works.

Your LDflags are setting `github.com/projectcalico/node/buildinfo`, but here you are passing in values from `github.com/projectcalico/felix/buildinfo`

You need to add a new package to node called `buildinfo`, import it, and pass values from it to `felix.Run`

Reply:
Fixed